ddsrecorder/src/cpp/user_interface/arguments_configuration.cpp

```diff
@@ -262,23 +257,26 @@ ProcessReturnCode parse_arguments(
                     break;
 
                 case optionIndex::ACTIVATE_DEBUG:
-                    activateDebug = true;
-                    log_filter_parse = "DDSRECORDER";
-                    log_verbosity_parse = eprosima::fastdds::dds::Log::Kind::Info;
+                    log_filter.set_value({
+                        {fastdds::dds::Log::Kind::Error, "DDSRECORDER"},
```

Comment:
Please make the error filter be "" and the warning filter be `"(DDSRECORDER|DDSPIPE)"` by default.